GameData/RealismOverhaul/RO_SAFConfig.cfg

```diff
@@ -0,0 +1,90 @@
+/// Example config - Replace 'MyPartWithSAFModule' with your part name
+/// @PART[MyPartWithSAFModule]:FOR[RealismOverhaul]
+/// {
+/// 	@MODULE[ModuleSimpleAdjustableFairing]
+/// 	{
+/// 		ROSAFRescale = 1.81 //Make everything 1.81 times bigger
+/// 	}
+/// }
+
+@PART[*]:HAS[@MODULE[ModuleSimpleAdjustableFairing]:HAS[#ROSAFRescale]]:AFTER[RealismOverhaul]
+{
+	@MODULE[ModuleSimpleAdjustableFairing]
+	{
+		&scale = 1
+		@scale *= #$ROSAFRescale$
+		@segmentLength *= #$ROSAFRescale$
+		@shieldingCenter[*] *= #$ROSAFRescale$
+		@shieldingBaseRadius *= #$ROSAFRescale$
+		@editorOpenOffset[*] *= #$ROSAFRescale$
+
+		@WALL_BASE
+		{
+			@CoM[*] *= #$../ROSAFRescale$
+			@rootOffset[*] *= #$../ROSAFRescale$
+		}
+
+		@WALL
+		{
+			@CoM[*] *= #$../ROSAFRescale$
+			@rootOffset[*] *= #$../ROSAFRescale$
+		}
+
+		@CONE
+		{
+			@CoM[*] *= #$../ROSAFRescale$
+			@rootOffset[*] *= #$../ROSAFRescale$
+		}
+
+		@CAP
+		{
+			@CoM[*] *= #$../ROSAFRescale$
+			@rootOffset[*] *= #$../ROSAFRescale$
+		}
+	}
+
+	@MODULE[ModuleB9PartSwitch]:HAS[#moduleID[fairingSwitch]]
```

Comment:
This `HAS` is fragile. It might work for most BDB things, but not always. I guess you can match on having subtypes with SAF switches inside.

Reply:
Removed `HAS`